skyportal/handlers/api/galaxy.py

```diff
@@ -0,0 +1,104 @@
+from baselayer.app.access import permissions, auth_or_token
+
+import astropy.units as u
+import healpix_alchemy as ha
+
+from ..base import BaseHandler
+from ...models import DBSession, Galaxy
+
+
+class GalaxyCatalogHandler(BaseHandler):
+    @permissions(['System admin'])
+    async def post(self):
+        """
+        ---
+        description: Ingest a Galaxy catalog
+        tags:
+          - galaxies
+        requestBody:
+          content:
+            application/json:
+              schema: GalaxyHandlerPost
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+
+        data = self.get_json()
+        catalog_name = data.get('catalog_name')
```

Comment:
What if these are not specified?

Reply:
@stefanv adding an error if this occurs.